<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="gotoscreen2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Producerand Comsumer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gotoChapter2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Chapter 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gotoChapter4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Chapter 4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RoundRobin">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Round Robin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RMA">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>RMA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EDF">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>EDF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Minimum_Laxity_First">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Minimum_Laxity_First</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DMA">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>DMA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Test">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FIFO">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>FIFO</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
